src/api/roles/client/fitness.py

```
@@ -0,0 +1,20 @@
+from sqlmodel import Session, select
+
+from src.database.session import get_session
+from src.database.account.models import Account
+from src.api.dependencies import get_client_account, PaginationParams
+from src.database.client.models import ClientWorkoutPlan
+
+router = APIRouter(prefix="/roles/client/fitness", tags=["client", "fitness"])
+
+@router.get("/query/plans")
+def query_client_workout_plans(
+    pagination: PaginationParams = Depends(PaginationParams),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)